### PR TITLE
ECDC-2584: Laser detector connection status

### PR DESCRIPTION
**Pull request by ebadkamil**
This PR is for the Laser detector specific fixes we did for ymir demo.

PS: Not pushing to Gerrit, since `pva` module has not made to Gerrit.

### CODE REVIEW

File: nicos_ess/ymir/devices/laser_detector.py [hunk 1]
```diff
@@ -62,20 +53,16 @@ def doRead(self, maxage=0):
         return [self.answer]
 
     def doFinish(self):
-        self._stop_processing()
-
-    def _stop_processing(self):
-        self.curstatus = status.OK, ""
+        pass
```
Comment (kmurica) [inline nicos_ess/ymir/devices/laser_detector.py:56]: I am thinking, maybe somehow we should notify the user these pass methods are not implemented? I am just worried that if it is quiet, people will expect it to work. What do you think?

Comment (ebadkamil) [reply nicos_ess/ymir/devices/laser_detector.py:56]: As of now these methods were  used to update the `curstatus`, which was eventually used in `doStatus`. Now the status we get directly from attached laser. There is another ticket https://jira.esss.lu.se/browse/ECDC-2603 where one may have to override `doSetPreset` and `doFinish` methods.  So for now, I guess pass should be sufficient. @mattclarke may comment.

Comment (kmurica) [reply nicos_ess/ymir/devices/laser_detector.py:56]: Fine by me then.

**Review by kmurica — APPROVED** [review]
LGTM

File: nicos_ess/ymir/setups/laser_detector.py [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+    laser_pv=device(
+        'nicos_ess.devices.epics.pva.EpicsStringReadable',
+        description='The laser PV',
+        readpv='plc-ts:PLC:slowdata',
+        lowlevel=True,
+        epicstimeout=3.0,
+    ),
     laser=device('nicos_ess.ymir.devices.laser_detector.LaserDetector',
-                 description='Laser detector in YMIR cave',),
+                 description='Laser detector in YMIR cave',
```
[the remark this answers is not in the document]
Comment (mattclarke) [reply nicos_ess/ymir/setups/laser_detector.py:12]: Ah, just a github rendering fail

**Pull request merged**